app/controllers/users_controller.rb

```diff
@@ -1,7 +1,99 @@
 class UsersController < ApplicationController
+  before_action :set_user, only: [:show, :edit, :update, :destroy, :trim, :massage_and_trim, :shave]
 
-  def list
+  # GET /users
+  # GET /users.json
+  def index
     @users = User.all
   end
-  alias :index :list
+
+  # GET /users/1
+  # GET /users/1.json
+  def show
+  end
+
+  # GET /users/new
+  def new
+    @user = User.new
+  end
+
+  # GET /users/1/edit
+  def edit
+  end
+
+  # POST /users
+  # POST /users.json
+  def create
+    @user = User.new(user_params)
+
+    respond_to do |format|
```

Comment:
I'll surely change the code in the users controller to respond_with where ever needed.